cmd/kyma/apply/function/function.go

```diff
@@ -64,7 +64,11 @@ Use the flags to specify the desired location for the source files or run the co
 
 func (c *command) Run() error {
 	if c.opts.Filename == "" {
-		c.opts.Filename = defaultFilename()
+		filename, err := defaultFilename()
+		if err != nil {
+			return err
```

Comment:
Could you please return the wrapped err fmt.Errorf(..)? 
Also applied for the rest